Snabble/Core/API/SnabbleAPI.swift

```diff
@@ -46,20 +46,43 @@ public struct SnabbleAPIConfig {
     // SQL statements that are executed just before the product database is opened
     public var initialSQL: [String]?
 
-    public init(appId: String, baseUrl: String, secret: String) {
+
+    /// Initialize the configuration for Snabble
+    /// - Parameters:
+    ///   - appId: Provide your personal `appId`
+    ///   - secret: The secret matching your `appId`
+    ///   - environment: Choose an environment you want to use
+    public init(appId: String, secret: String, environment: SnabbleAPI.Environment = .production) {
         self.appId = appId
-        self.baseUrl = baseUrl
+        self.environment = environment
         self.secret = secret
     }
 
-    static let none = SnabbleAPIConfig(appId: "none", baseUrl: "", secret: "")
+    static let none = SnabbleAPIConfig(appId: "none", secret: "")
 }
 
 public extension Notification.Name {
     static var metadataLoaded = Notification.Name(rawValue: "io.snabble.metadataLoaded")
 }
 
 public enum SnabbleAPI {
+    public enum Environment {
+        case test
+        case stage
+        case production
+
+        var urlString: String {
```

Comment:
Musss IMO auch public sein, damit Code von aussen (zB aus unseren QR-Code-Umschaltern) den korrekten Case ermitteln kann

Reply:
Kannst du mir sagen wo ich den Teil genau finde worauf du hinweist? Ich finde es auf Anhieb nicht.

Reply:
477ad0e5cfb231d2cf912e3c5f8b32b41f3b0878

Reply:
In der App, SnabbleConfig.swift:36

Reply:
@gereons könntest du [hier](https://github.com/snabble/iOS-App/pull/238) die Anpassungen hinzufügen? Ich sehe die Notwendigkeit leider nicht.

Reply:
Siehe Kommentar da - da werden nochmal alle URLs wiederholt, ich würde da eine Schleife benutzen und dazu hier den Enum `CaseIterable` machen

Reply:
Finde ich eine gute Idee, dazu habe ich noch `Equatable` hinzugefügt.

Reply:
5cefeff3b93bd179145460c47e4ca78ba80db09d